### PR TITLE
blog: Add short post redirecting to the Atix Deb blog post

### PR DESCRIPTION
**Pull request by GregSutcliffe**
Just a quick post to highlight the awesome work Atix are up to. @mdellweg, how's this look? :)

### CODE REVIEW

File: _posts/2017-10-23-atixag-shows-off-support-for-apt.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: AtixAG shows off support for Apt repos in Katello
```
Comment (mdellweg) [inline _posts/2017-10-23-atixag-shows-off-support-for-apt.md:3]: I'd like to see a space as in "Atix AG" here. And further down...

File: _posts/2017-10-23-atixag-shows-off-support-for-apt.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+date: 2017-10-23 14:57:34
+author: Greg SUtcliffe
```
Comment (mdellweg) [inline _posts/2017-10-23-atixag-shows-off-support-for-apt.md:5]: I guess the capital 'U' is by mistake?

**Review by ehelms — APPROVED** [review]

**Review by mdellweg — COMMENTED** [review]
Sounds good! Just two... let's call them typos.

File: _posts/2017-10-23-atixag-shows-off-support-for-apt.md (hunk 1 above)
Comment (GregSutcliffe) [reply _posts/2017-10-23-atixag-shows-off-support-for-apt.md:3]: Fixed, thanks.

File: _posts/2017-10-23-atixag-shows-off-support-for-apt.md (hunk 2 above)
Comment (GregSutcliffe) [reply _posts/2017-10-23-atixag-shows-off-support-for-apt.md:5]: I'm (case) insensitive :D - thanks :P

**Comment by GregSutcliffe** [comment]
Thanks @mdellweg - pushed as 43f646856 since @ehelms has given the +1.

**Pull request closed without merging**